addon/components/md-nav.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import layout from '../templates/components/md-nav';
+
+const { run: { next }, Component } = Ember;
+
+export default Component.extend({
+  classNames: ['md-nav'],
+  layout,
+  tagName: 'nav',
```

Comment:
Because this starts out as a `nav` element, it doesn't seem to be possible to apply the [`navbar-fixed`](http://materializecss.com/navbar.html#navbar-fixed) class as part of the component.  

Also wrapping the component in the class does not work, e.g.:

```hbs
<div class="navbar-fixed">
  {{md-nav}}
</div>
```